sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -66,8 +65,7 @@ const (
 	URNLegacyProgressReporting = "beam:protocol:progress_reporting:v0"
 	URNMultiCore               = "beam:protocol:multi_core_bundle_processing:v1"
 
-	URNRequiresSplittableDoFn     = "beam:requirement:pardo:splittable_dofn:v1"
-	URNRequiresBundleFinalization = "beam:requirement:pardo:finalization:v1"
+	URNRequiresSplittableDoFn = "beam:requirement:pardo:splittable_dofn:v1"
```

Comment:
This looks like a bad merge with my bundle finalization changes

Reply:
Ah whoops!
I spent all my effort making sure the proto didn't get clobbered, I forgot about the rest of the change. Thanks!